src/components/App.js

```diff
@@ -6,7 +6,10 @@ import DetailPage from '../pages/DetailPage'
 import HomePage from '../pages/HomePage'
 import createUrlQuery from '../services/createUrlQuery'
 import getFilters from '../services/getFilters'
-require('dotenv').config()
+import SavedRecipes from '../pages/SavedRecipes'
+import saveRecipes from '../lib/saveRecipes'
```

Comment:
If you might want to use this functions later on in a different context you could name them in a more generic way, eg. 'saveToLocal' and 'loadFromLocal'